code/modules/urist/mob/spoopymobs.dm

```diff
@@ -21,11 +21,131 @@
 	move_to_delay = 7
 	stat_attack = 1
 	ranged = 0
-	universal_speak = 1
 	var/plague = 0 //whether biting dead people spawns new zombies
 	var/regen = 0 //if true, they won't stay down, but revive after a delay
 	var/regen_delay = 900 //delay for regen revive
 
+/datum/reagent/toxin/zombie/uristzombie
+	flags = AFFECTS_DEAD
```

Comment:
This flag doesn't appear to function, transformations never progressed after death.

Reply:
They do, but not because of this flag; I think brainmed broke this flag -_-